data/en/gettimezoneinfo.json

```diff
@@ -6,7 +6,8 @@
 	"related":["getTimezone","setTimezone"],
 	"description":" Gets local time zone information for the computer on which it\n is called, relative to Universal Time Coordinated (UTC). UTC is\n the mean solar time of the meridian of Greenwich, England.",
 	"params": [
-		{"name":"text","description":"","required":true,"default":"","type":"string","values":[]}
+		{"name":"timezone","description":"Lucee Specify a timezone to return info from","required":true,"default":"","type":"string","values":[]},
+		{"name":"locale", "description":"Lucee Locale used to format the output","required":true,"default":"","type":"string","values":[]}
```

Comment:
If you change "Lucee" to "Lucee4.5+" (first Lucee version = always implemented), it'll get the label.